.licenserc.yaml

```
@@ -0,0 +1,48 @@
+# ********************************************************************************
+# Copyright (c) 2024 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made
+# available under the terms of the Apache Software License 2.0
+# which is available at https://www.apache.org/licenses/LICENSE-2.0.
+#
+# SPDX-License-Identifier: Apache-2.0
+# ********************************************************************************
+
+header:
+  license:
+    spdx-id: Apache-2.0
+    copyright-owner: Eclipse Foundation
+    content: |
+      ********************************************************************************
+      Copyright (c) [year] Contributors to the [owner]
+
+      See the NOTICE file(s) with this work for additional
+      information regarding copyright ownership.
+      
+      This program and the accompanying materials are made
+      available under the terms of the Apache Software License 2.0
+      which is available at https://www.apache.org/licenses/LICENSE-2.0.
+      
+      SPDX-License-Identifier: Apache-2.0
+      ********************************************************************************
+    pattern: Copyright (c) \d{4} Contributors to the Eclipse Foundation
+      
+  comment: on-failure
+
+  paths-ignore:
+    - 'LICENSE'
+    - 'NOTICE'
+    - '**/*.md'
+    - '.gitignore'
+    - '.dockerignore'
+    - 'docker/.gitignore'
+    - 'security/.gitignore'
+    - 'security/mk-ca-bundle.pl'
+    - 'security/certdata.txt'
+    - 'sbin/*.template'
```

Don't claim on third-party public keys.

```suggestion
    - 'sbin/*.template'
    -'sbin/sig_check/*.asc'
```

---

I see you have gone for ignoring all keys rather than third party only. That will be OK for the moment as we don't use our keys in there.

---

yeah it feels like the sane thing to do for now. We can always change it later